Makefile

```diff
@@ -1,14 +1,21 @@
+# Default shell
+SHELL := /bin/bash
+
 # Configuration
-PROJECT ?= blog
+HUGO_VERSION := 0.30.2
+PORT ?= 8484
+BASE_URL ?= "//localhost:$(PORT)/"
```

Comment:
Why port has changed?

Reply:
Initially it was `1313`, who is the hugo default. Since we can have many hugo services already running, I just tried to define a new default config not having conflicts with our other projects.
Is that wrong?

Reply:
hugoInitially it was `1313`, who is the hugo default. Since we can have many hugo services already running, I just tried to define a new default config not having conflicts with our other projects.
Is that wrong?

Reply:
No, it's not. It's just a change that is not evident from the title description, so it's difficult to understand why it's needed